pyvista/plotting/camera.py

```diff
@@ -150,11 +150,6 @@ def model_transform_matrix(self, matrix):
         vtk_matrix.DeepCopy(matrix.ravel())
         self.SetModelTransformMatrix(vtk_matrix)
 
-    @property
-    def is_parallel_projection(self):
-        """Return True if parallel projection is set."""
-        return self._is_parallel_projection
-
```

Comment:
How about raise [DeprecationWarning](https://docs.python.org/3/library/exceptions.html#DeprecationWarning) about `is_parallel_projection` ?

Reply:
Yes, could you add that?

Reply:
Done in https://github.com/pyvista/pyvista/pull/1457/commits/0b2a8cf0664ac0f89f621d4da42bc71593cda00b